Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4B9Psk2XgowbhPf3YaoofcjnCEzxPj3KmUkXJb9c+rI0AAAApB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAz+EgYp/eT83ILSktQi/cTSknxdoK6SovycHCAfANhVmRrPAAAA[0mauto-controller in /root/.jenkins/workspace/test-nic-simple-Test
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (set args)
[8mha:////4KpTcsSlLG7yEvo1vGv2QG1z86HMYkLASFXVuKwhd7SZAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaWGsbAmNJ0AWEZb8zwLrbuWJvJp3kLiJlZNMMm+a93rDOic4UbLcG+wdZu14DKOti0+U+lugiXu6ck2YKRguzSSpM+cFJRUDS1gDKwEbgzpQdmgLbIVXD9UGhba9lFS/o4DGdQM8gYlqLiqVL8wJdvexy4Q/z18BzLEA29ce4gfg7KmOvAAAAA==[0m[Pipeline] sh
+ python /root/.jenkins/workspace/test-nic-simple-Test/npxtester/conf/change_args.py '{"sut_conf":{"host":"127.0.0.1","connect_switch_sut_pf":"ens801f0","pf1":"ens801f0","os":"rehat"},"dhcp":{"ipv4":"127.0.0.1","ipv6":"::1"}}'
[8mha:////4ChDQWb8Lvr0RoXQ1L8EY4p37+Po0Q8Mb8bchs8JeqDJAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbGRNjZUtoPAGyiLDkfxZYdytP5NW8g8RNrJxkknnTvNcb1jnBiZLl3mDvMGvHYxhtXXyi1N8CTdzTlWvCTMFwaSZJnTkvKKkYWMIaWAnYGNSBskNbYCu8eqg2KLTtpaT6HQU0rhvgCUxUc1GpfGFOsLuPXSb8ef4KYI6xADvU7j9Dg2gqvAAAAA==[0m[Pipeline] }
[8mha:////4J0p4zt/+On27QitkvU5TFB8M2wwWYWaAHIuYiUer4fkAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQRbWRhjZUtoPAGyiLDkfxZYdytP5NW8g8RNrJxkknnTvNcb1jnBiZLl3mDvMGvHYxhtXXyi1N8CTdzTlWvCTMFwaSZJnTkvKKkYWMIaWAnYGNSBskNbYCu8eqg2KLTtpaT6HQU0rhvgCUxUc1GpfGFOsLuPXSb8ef4KYI6xADvU7j9J+wGOvAAAAA==[0m[Pipeline] // stage
[8mha:////4Ea3/4ZYuTM98yMZ9UntzMXOyQ2ItSOm6XAUUO4QqkITAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQDEh2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INtuQqUcEAAAA=[0m[Pipeline] stage
[8mha:////4IJd0qxfNFkcgwUT9OFke9rxaJjwCH2G9mHJ+5FMmbLfAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycoiUzoiJNerCCUITQtLo/zZJSSdOxNW4Ay2VmPBg2Zas93pDGQOcKBjuNHYWY2t570czJ54pdDdPmTu68pYwkte80bkhpc9rbShpWFUw2AjYamw9RYsmwU44+ZCVl2iqSwrzdhTArBrgCYWYyUmG9C1TgP19VJHwx/kLgKlPwOrDYvXyLD8BobDcwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4KACpsxoNqug+yeGIE2K55MD+IyPDLfbYOrean7h1Zl/AAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggysXWsbBkaT4AJIoT5nwCRVJ7Iq3kH0cxYucXOvm3e8wXrGOBIQTOrcDAYO8O8m3RZLFMYro4ys3RhHWEkp5hQWVCvTgsKSgqWVDWsOGwUdo6iQZ1gy628y8ZJ1M05hfIdONSmH+EBFS/mJEP6whxgd5v6SPjz/BXA7BPUbfupvX8DpQJrNb4AAAA=[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics -s -m nicSimple --html=/root/testconfig/testlogs/HtmlReport/nic_simple_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 10 items / 7 deselected / 3 selected

npxtester/Tests/testNics/test_nics.py 2021-12-03 12:51:44 - NicTestFixture.py - [line:28] - root - INFO - Starting to run test_silver_Networking_ConnectivityEnableDisable
2021-12-03 12:51:44 - NicTestFixture.py - [line:29] - root - INFO - setup log for testing in the fixture class fbefore
2021-12-03 12:51:44 - test_nics.py - [line:18] - root - INFO - this is Test_Nics setup.
2021-12-03 12:51:44 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2021-12-03 12:51:44 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2021-12-03 12:51:44 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 127.0.0.1
2021-12-03 12:51:47 - sshInstance.py - [line:44] - root - INFO - echo 0 > /sys/class/net/ens801f0/device/sriov_numvfs
2021-12-03 12:51:47 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:51:52 - test_nics.py - [line:37] - root - INFO - run silver_Networking_ConnectivityEnableDisable
2021-12-03 12:51:56 - sshInstance.py - [line:69] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Dec  3 10:12:54 2021 from 10.238.10.192
[root@~]#ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.029 ms
2021-12-03 12:51:57 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
2021-12-03 12:51:58 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.035 ms
2021-12-03 12:51:59 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.041 ms
2021-12-03 12:52:00 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.035 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4072ms
rtt min/avg/max/mdev = 0.029/0.035/0.041/0.003 ms
[root@~]#
2021-12-03 12:52:08 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 down
[root@~]#
2021-12-03 12:52:16 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 up
[root@~]#
2021-12-03 12:52:23 - sshInstance.py - [line:69] - root - INFO - ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.039 ms
2021-12-03 12:52:24 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
2021-12-03 12:52:25 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.027 ms
2021-12-03 12:52:26 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.034 ms
2021-12-03 12:52:27 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.035 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4135ms
rtt min/avg/max/mdev = 0.027/0.034/0.039/0.004 ms
[root@~]#
2021-12-03 12:52:35 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 down
[root@~]#
2021-12-03 12:52:43 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 up
[root@~]#
2021-12-03 12:52:50 - sshInstance.py - [line:69] - root - INFO - ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.039 ms
2021-12-03 12:52:51 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.036 ms
2021-12-03 12:52:52 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.035 ms
2021-12-03 12:52:53 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.035 ms
2021-12-03 12:52:54 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.036 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4111ms
rtt min/avg/max/mdev = 0.035/0.036/0.039/0.004 ms
[root@~]#
2021-12-03 12:53:02 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 down
[root@~]#
2021-12-03 12:53:10 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 up
[root@~]#
2021-12-03 12:53:17 - sshInstance.py - [line:69] - root - INFO - ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.025 ms
2021-12-03 12:53:18 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
2021-12-03 12:53:19 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.038 ms
2021-12-03 12:53:20 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.034 ms
2021-12-03 12:53:21 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.034 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4088ms
rtt min/avg/max/mdev = 0.025/0.033/0.038/0.005 ms
[root@~]#
2021-12-03 12:53:29 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 down
[root@~]#
2021-12-03 12:53:37 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 up
[root@~]#
2021-12-03 12:53:44 - sshInstance.py - [line:69] - root - INFO - ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.020 ms
2021-12-03 12:53:45 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
2021-12-03 12:53:46 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.034 ms
2021-12-03 12:53:47 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.035 ms
2021-12-03 12:53:48 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.036 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4129ms
rtt min/avg/max/mdev = 0.020/0.032/0.036/0.006 ms
[root@~]#
2021-12-03 12:53:56 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 down
[root@~]#
2021-12-03 12:54:04 - sshInstance.py - [line:89] - root - INFO - ifconfig ens801f0 up
[root@~]#
2021-12-03 12:54:11 - sshInstance.py - [line:69] - root - INFO - ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.024 ms
2021-12-03 12:54:12 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.034 ms
2021-12-03 12:54:13 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.035 ms
2021-12-03 12:54:14 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.035 ms
2021-12-03 12:54:15 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.034 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4105ms
rtt min/avg/max/mdev = 0.024/0.032/0.035/0.006 ms
[root@~]#
.2021-12-03 12:54:18 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2021-12-03 12:54:18 - test_nics.py - [line:30] - root - INFO - this is Test_Nics teardown
2021-12-03 12:54:18 - NicTestFixture.py - [line:28] - root - INFO - Starting to run test_silver_ConnectivityReboot
2021-12-03 12:54:18 - NicTestFixture.py - [line:29] - root - INFO - setup log for testing in the fixture class fbefore
2021-12-03 12:54:18 - test_nics.py - [line:18] - root - INFO - this is Test_Nics setup.
2021-12-03 12:54:18 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2021-12-03 12:54:18 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2021-12-03 12:54:18 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 127.0.0.1
2021-12-03 12:54:21 - sshInstance.py - [line:44] - root - INFO - echo 0 > /sys/class/net/ens801f0/device/sriov_numvfs
2021-12-03 12:54:22 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:54:27 - test_nics.py - [line:60] - root - INFO - run silver_ConnectivityReboot
2021-12-03 12:54:27 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2021-12-03 12:54:27 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2021-12-03 12:54:27 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 127.0.0.1
2021-12-03 12:54:31 - sshInstance.py - [line:69] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Dec  3 12:54:18 2021 from 127.0.0.1
[root@~]#ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.046 ms
2021-12-03 12:54:32 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
2021-12-03 12:54:33 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.035 ms
2021-12-03 12:54:34 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.037 ms
2021-12-03 12:54:35 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.036 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4112ms
rtt min/avg/max/mdev = 0.035/0.037/0.046/0.008 ms
[root@~]#
2021-12-03 12:54:42 - sshInstance.py - [line:69] - root - INFO - ping6 -c 5 -I ens801f0 ::1
connect: Network is unreachable
[root@~]#
F2021-12-03 12:54:45 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2021-12-03 12:54:45 - test_nics.py - [line:30] - root - INFO - this is Test_Nics teardown
2021-12-03 12:54:45 - NicTestFixture.py - [line:28] - root - INFO - Starting to run test_gold_PI_Networking_SRIOV_CheckDevice
2021-12-03 12:54:45 - NicTestFixture.py - [line:29] - root - INFO - setup log for testing in the fixture class fbefore
2021-12-03 12:54:45 - test_nics.py - [line:18] - root - INFO - this is Test_Nics setup.
2021-12-03 12:54:45 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2021-12-03 12:54:45 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2021-12-03 12:54:45 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 127.0.0.1
2021-12-03 12:54:48 - sshInstance.py - [line:44] - root - INFO - echo 0 > /sys/class/net/ens801f0/device/sriov_numvfs
2021-12-03 12:54:48 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:54:53 - test_nics.py - [line:79] - root - INFO - run gold_PI_Networking_SRIOV_CheckDevice
2021-12-03 12:54:56 - sshInstance.py - [line:44] - root - INFO - echo 4 > /sys/class/net/ens801f0/device/sriov_numvfs
2021-12-03 12:54:56 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:55:04 - sshInstance.py - [line:44] - root - INFO - ifconfig ens801f0v0 192.168.10.11
2021-12-03 12:55:04 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:55:12 - sshInstance.py - [line:44] - root - INFO - ifconfig ens801f0v1 192.168.10.12
2021-12-03 12:55:12 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:55:20 - sshInstance.py - [line:44] - root - INFO - ifconfig ens801f0v2 192.168.10.13
2021-12-03 12:55:20 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:55:28 - sshInstance.py - [line:44] - root - INFO - ifconfig ens801f0v3 192.168.10.14
2021-12-03 12:55:28 - sshInstance.py - [line:48] - root - INFO - 
2021-12-03 12:55:36 - sshInstance.py - [line:44] - root - INFO - ping -c 5 127.0.0.1
2021-12-03 12:55:41 - sshInstance.py - [line:48] - root - INFO - PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.023 ms
64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.036 ms
64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.034 ms
64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.035 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4087ms
rtt min/avg/max/mdev = 0.023/0.032/0.036/0.007 ms

2021-12-03 12:55:49 - sshInstance.py - [line:44] - root - INFO - ping6 -c 5 -I ens801f0v0 ::1
2021-12-03 12:55:49 - sshInstance.py - [line:48] - root - INFO - 
F2021-12-03 12:55:54 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2021-12-03 12:55:54 - test_nics.py - [line:30] - root - INFO - this is Test_Nics teardown


=================================== FAILURES ===================================
___________________ Test_Nics.test_silver_ConnectivityReboot ___________________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f7162d80b00>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicSimple
    @pytest.mark.nicRegression
    def test_silver_ConnectivityReboot(self):
        logs = Logger()
        logging.info("run silver_ConnectivityReboot")
        rs = 0
        cmdip = "ping -c 5 {}\n".format(self.dhcp["ipv4"])
        cmdipv6 = "ping6 -c 5 -I {} {}\n".format(self.sut_conf["connect_switch_sut_pf"],self.dhcp["ipv6"])
        for num in range(5):
            cn_test_sut = sshInstance()
            client, session = cn_test_sut.get_ssh_instance(self.sut_conf["host"], self.sut_conf["user"],
                                                           self.sut_conf["password"], self.sut_conf["port"])
            output, stderr = cn_test_sut.run_ssh_cmd(session, cmdip, self.check_str)
            assert 200 == cn_test_sut.commond_rs_check(output, "0% packet loss", True)
            output, stderr = cn_test_sut.run_ssh_cmd(session, cmdipv6, self.check_str)
>           assert 200 == cn_test_sut.commond_rs_check(output, "0% packet loss", True)
E           AssertionError: assert 200 == 400
E            +  where 400 = <bound method sshInstance.commond_rs_check of <libs.commons.sshInstance.sshInstance object at 0x7f7166566eb8>>('ping6 -c 5 -I ens801f0 ::1\r\nconnect: Network is unreachable\r\n[root@~]#', '0% packet loss', True)
E            +    where <bound method sshInstance.commond_rs_check of <libs.commons.sshInstance.sshInstance object at 0x7f7166566eb8>> = <libs.commons.sshInstance.sshInstance object at 0x7f7166566eb8>.commond_rs_check

npxtester/Tests/testNics/test_nics.py:71: AssertionError
------------------------------ Captured log call -------------------------------
INFO     root:test_nics.py:60 run silver_ConnectivityReboot
INFO     paramiko.transport:transport.py:1819 Connected (version 2.0, client OpenSSH_8.0)
INFO     paramiko.transport:transport.py:1819 Authentication (password) successful!
INFO     root:sshInstance.py:28 SSH on SUT: 127.0.0.1
INFO     root:sshInstance.py:69 Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Dec  3 12:54:18 2021 from 127.0.0.1
[root@~]#ping -c 5 127.0.0.1
PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.046 ms
INFO     root:sshInstance.py:69 64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
INFO     root:sshInstance.py:69 64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.035 ms
INFO     root:sshInstance.py:69 64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.037 ms
INFO     root:sshInstance.py:69 64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.036 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4112ms
rtt min/avg/max/mdev = 0.035/0.037/0.046/0.008 ms
[root@~]#
INFO     root:sshInstance.py:69 ping6 -c 5 -I ens801f0 ::1
connect: Network is unreachable
[root@~]#
_____________ Test_Nics.test_gold_PI_Networking_SRIOV_CheckDevice ______________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f7162d76240>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicSimple
    @pytest.mark.nicRegression
    def test_gold_PI_Networking_SRIOV_CheckDevice(self):
        logging.info("run gold_PI_Networking_SRIOV_CheckDevice")
        cmd = "echo 4 > /sys/class/net/{}/device/sriov_numvfs".format(self.sut_conf["connect_switch_sut_pf"])
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, cmd)
        if stdout or stderr:
            assert 200 == 400
        manual_ip_cmd = "ifconfig {}{} 192.168.10.{}"
        for i in range(4):
            vf = "v"+str(i)
            ip_part = str(11+i)
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, manual_ip_cmd.format(self.sut_conf["pf1"], vf, ip_part))
            if stdout or stderr:
                assert 200 == 400
        ping_ipv4_cmd = "ping -c 5 {}"
        ping_ipv6_cmd = "ping6 -c 5 -I {}{} {}"
        for i in range(4):
            vf = "v"+str(i)
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, ping_ipv4_cmd.format(self.dhcp["ipv4"]))
            if stdout.find("0% packet loss") == -1:
                assert 200 == 400
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, ping_ipv6_cmd.format(self.sut_conf["pf1"], vf, self.dhcp["ipv6"]))
            if stdout.find("0% packet loss") == -1:
>               assert 200 == 400
E               assert 200 == 400

npxtester/Tests/testNics/test_nics.py:100: AssertionError
------------------------------ Captured log call -------------------------------
INFO     root:test_nics.py:79 run gold_PI_Networking_SRIOV_CheckDevice
INFO     root:sshInstance.py:44 echo 4 > /sys/class/net/ens801f0/device/sriov_numvfs
INFO     root:sshInstance.py:48 
INFO     root:sshInstance.py:44 ifconfig ens801f0v0 192.168.10.11
INFO     root:sshInstance.py:48 
INFO     root:sshInstance.py:44 ifconfig ens801f0v1 192.168.10.12
INFO     root:sshInstance.py:48 
INFO     root:sshInstance.py:44 ifconfig ens801f0v2 192.168.10.13
INFO     root:sshInstance.py:48 
INFO     root:sshInstance.py:44 ifconfig ens801f0v3 192.168.10.14
INFO     root:sshInstance.py:48 
INFO     root:sshInstance.py:44 ping -c 5 127.0.0.1
INFO     root:sshInstance.py:48 PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.
64 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.023 ms
64 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.035 ms
64 bytes from 127.0.0.1: icmp_seq=3 ttl=64 time=0.036 ms
64 bytes from 127.0.0.1: icmp_seq=4 ttl=64 time=0.034 ms
64 bytes from 127.0.0.1: icmp_seq=5 ttl=64 time=0.035 ms

--- 127.0.0.1 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4087ms
rtt min/avg/max/mdev = 0.023/0.032/0.036/0.007 ms

INFO     root:sshInstance.py:44 ping6 -c 5 -I ens801f0v0 ::1
INFO     root:sshInstance.py:48
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:15
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_iperf_client_4hour.py:15: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:44
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_iperf_client_4hour.py:44: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_network_sriov_stress.py:344
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_network_sriov_stress.py:344: PytestUnknownMarkWarning: Unknown pytest.mark.nicRegression - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicRegression

npxtester/Tests/testNics/test_network_sriov_stress.py:345
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_network_sriov_stress.py:345: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:457
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_network_sriov_stress.py:457: PytestUnknownMarkWarning: Unknown pytest.mark.nicRegression - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicRegression

npxtester/Tests/testNics/test_network_sriov_stress.py:458
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_network_sriov_stress.py:458: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_nics.py:34
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:34: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:35
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:35: PytestUnknownMarkWarning: Unknown pytest.mark.nicRegression - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicRegression

npxtester/Tests/testNics/test_nics.py:56
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:56: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:57
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:57: PytestUnknownMarkWarning: Unknown pytest.mark.nicRegression - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicRegression

npxtester/Tests/testNics/test_nics.py:76
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:76: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:77
  /root/.jenkins/workspace/test-nic-simple-Test/npxtester/Tests/testNics/test_nics.py:77: PytestUnknownMarkWarning: Unknown pytest.mark.nicRegression - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicRegression

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_simple_case_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_silver_ConnectivityReboot
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_gold_PI_Networking_SRIOV_CheckDevice
====== 2 failed, 1 passed, 7 deselected, 12 warnings in 249.83s (0:04:09) ======
[8mha:////4KMKnj+jwomtBZRlOM1zDNsT7NVP2UylJUBAmwQdpqOFAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyGUvHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzN4nqNv9p9o3geV5574AAAA=[0m[Pipeline] }
[8mha:////4PHbOgS4fTQzlO31aLQygq6Gmyy7LbpReNAbbgHuNOsPAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICEh2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwhQ7nbf2r7Bq/le42+AAAA[0m[Pipeline] // stage
[8mha:////4I9CBL1TQQZuqbZ2jRHSmcHnuMbbRTwUd9k6KAOagwtcAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRElSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsP9aSlGr0AAAA=[0m[Pipeline] }
[8mha:////4AgCtwjc4t5tcZsCa10f3l1mNPWqCUOes7yjbzZP2H6lAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRQqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AIZLYcO9AAAA[0m[Pipeline] // node
[8mha:////4HNhgbRMdr5fLy6tNmZPXDu18JtR1J6yjnVzc/IrJ0XIAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRNBSIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsPoFIGib0AAAA=[0m[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
